### ISSUE TITLE
Speaker Diarization with Marblenet and ClusterDiarizer issue

### ISSUE THREAD

**Issue by ddebnath228** [NVIDIA/NeMo#4535]
**Describe the bug**
I am not getting correct time stamps for speech segment and many speech chunks are removed. I am using pretrained Marblenet and speakerdiarization_speakernet models. It removes lots of speech data in between the chunks as below:

['0.0 0.23 speaker_1', '1.55 1.99 speaker_2', '3.18 4.25 speaker_2', '5.77 11.35 speaker_1',

**Steps/Code to reproduce bug**

from nemo.collections.asr.models import ClusteringDiarizer
sd_model = ClusteringDiarizer(cfg=cfg)
sd_model.diarize()

It generates RTTM file with speaker and timestamp which itself is wrong. Tried with multiple CFG parameter changes but no change in the output. 

**Expected behavior**

The chunks should be continous one after another so that no loss of actual speech data.

**Environment overview (please complete the following information)**

 - Environment location: Collab
 - Method of NeMo install: [pip install or from source]. PIP install from git as per the tutorials and git
 - If method of install is [Docker], NO

**Environment details**

If NVIDIA docker image is used you don't need to specify these.
Otherwise, please provide:
- OS version Windows 10
- PyTorch version 1.11
- Python version 3.7

**Additional context**

Example: GPU model

**Comment by ddebnath228** [comment]
[diarizationExample.zip](https://github.com/NVIDIA/NeMo/files/9078792/diarizationExample.zip)
Uploaded the Wav file too

**Comment by nithinraok** [comment]
VAD and Speaker Embedding extractor model are outdated. 
Please use [vad_telephony_marblenet](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/vad_telephony_marblenet) for VAD and [titanet_large](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/titanet_large) for Speaker Embedding extractor

**Comment by tango4j** [comment]
Seems like this is not a bug/issue in NeMo pipeline (changed to question), in the worst case this could be the limit of model performance.
